sourcegraph/jetbrains
# Add usage limit notification in subscription tab

It fixes https://github.com/sourcegraph/jetbrains/issues/147

## Test plan
* Login as user with exceeded rate limit
* Type a chat message
* Check if chat notification is present in subscription tab
* Trigger autocomplete
* Check if notification for chat and autocomplete is present in subscription tab
* Change user
* Again login as user with exceeded rate limit
* Trigger autocomplete 
* Check if autocomplete notification is present in subscription tab


https://github.com/sourcegraph/jetbrains/assets/9321940/78c0afed-8bf2-4ce7-b181-6fdf1ab24bf8


https://github.com/sourcegraph/jetbrains/assets/9321940/632ca94f-32aa-4f3c-98a9-01f682998362
